plugins/lua/sort/places.lua

```diff
@@ -33,9 +34,15 @@ local language_name_types = {
     [df.language_name_type.SymbolFood] = 'Inn Tavern',
     [df.language_name_type.Temple] = 'Temple',
     [df.language_name_type.Hospital] = 'Hospital',
-    [df.language_name_type.Guildhall] = 'Guildhall'
+    [df.language_name_type.Guildhall] = 'Guildhall',
+    [df.language_name_type.Library] = 'Library'
 }
 
+local function get_location_religion(religion_id, religion_type)
+    if religion_type == -1 then return 'Temple'
```

Comment:
```suggestion
    if religion_type == df.temple_deity_type.None then return 'Temple'
```

Reply:
could you possibly fix this in locationselector.lua in `get_religion_string` as well? `0` should be `df.temple_deity_type.Deity` and `1` should be `df.temple_deity_type.Religion`